pyxis/upload_rpm_data.py

```diff
@@ -181,6 +218,32 @@ def create_image_rpm_manifest(graphql_api: str, image_id: str, rpms: list) -> st
     return data["create_image_rpm_manifest"]["data"]["_id"]
 
 
+def patch_image_rpm_manifest(graphql_api: str, image_id: str) -> str:
+    """Patch ContainerImageRPMManifest object in Pyxis using GraphQL API with an empty body.
+
+    This is used to retrigger the sync on the Pyxis side when the manifest already exists
+    but the rpm_manifest field on the ContainerImage is null.
+    """
+    mutation = """
+mutation ($id: String!, $input: ContainerImageRPMManifestInput!) {
+    update_image_rpm_manifest(id: $id, input: $input) {
+        data {
+            _id
+        }
+        error {
+            detail
+        }
+    }
+}
+"""
+    variables = {"id": "konflux-" + image_id, "input": {}}
```

Comment:
Same here - This will throw error, the id must be an [ObjectId](https://www.mongodb.com/docs/manual/reference/bson-types/#std-label-objectid)

```suggestion
    variables = {"id": image_id, "input": {}}
```